### PR TITLE
:bug: removed additionalProperties

### PR DESCRIPTION
**Pull request by nessy**

### CODE REVIEW

File: specification/schemas/CarrierContract.json [hunk 1]
```diff
@@ -12,7 +12,6 @@
       "properties": {
         "attributes": {
           "type": "object",
-          "additionalProperties": false,
```
Comment (Shodske) [inline specification/schemas/CarrierContract.json]: Why is this removed?

Comment (NickVries) [reply specification/schemas/CarrierContract.json]: With post carrier contract, we want to specify that you can send credentials. So that's done with an `"allOf"` in the post path. But `"allOf"` is not possible when `"additionalProperties": false` is present.

File: specification/paths/CarrierContracts.json [hunk 2]
```diff
@@ -181,6 +192,17 @@
                             "currency",
                             "name"
                           ]
+                        },
+                        "relationships": {
+                          "properties": {
+                            "owner": {
+                              "example": {
+                                "links": {
```
Comment (Shodske) [inline specification/paths/CarrierContracts.json]: `links` shouldn't just be an example, it should be properly defined in a schema.

Comment (NickVries) [reply specification/paths/CarrierContracts.json]: The owner relationship is in this case a `"oneOf"` relationship. Which means that our UI doesn't automatically produce an example. If we put it in the example of the schema, that means that post also has links in the example. That's why it was put here as an extra for responses.
